godot-macros/src/docs.rs

```diff
@@ -70,22 +88,19 @@ pub fn make_inherent_impl_docs(
 }
 
 pub fn make_virtual_impl_docs(vmethods: &[ImplMember]) -> TokenStream {
-    match vmethods
+    let virtual_methods = vmethods
         .iter()
         .filter_map(|x| match x {
             venial::ImplMember::AssocFunction(f) => Some(f.clone()),
             _ => None,
         })
-        .map(make_virtual_method_docs)
-        .collect::<Option<String>>()
-    {
-        Some(vmethods) => quote! {
-            virtual_method_docs: #vmethods,
-        },
-        None => quote! {
-            virtual_method_docs: ""
-        },
-    }
+        .filter_map(make_virtual_method_docs)
+        .collect::<String>();
+    virtual_methods
+        .is_empty()
+        .not()
+        .then(|| quote! { virtual_method_docs: #virtual_methods.into(), })
+        .unwrap_or_else(|| quote! { virtual_method_docs: None, })
```

Comment:
why did you obfuscate the match?
```suggestion
    match virtual_methods {
        "" => quote! { virtual_method_docs: None, },
        _ => quote! { virtual_method_docs: #virtual_methods.into(), }
    }
```

Reply:
fixed

Reply:
why are you applying my suggestion weirdly? you seem to really want to use `is_empty`